content/en/docs/reference/command-line-tools-reference/feature-gates/strict-cost-enforcement-for-vap.md

```diff
@@ -0,0 +1,15 @@
+---
+title: StrictCostEnforcementForVAP
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: false
+    fromVersion: "1.30"
```

Comment:
If the fact is that the gate was cherry-picked to 1.30 release, we need to be specific about the version string. It was not there in `1.30.0`. Please clarify the exact patch version.

Reply:
I think this feature gate was new in v1.31.0
```suggestion
    fromVersion: "1.31"
```

Have I got that right?